test/storage.jl

```diff
@@ -92,27 +92,52 @@ import AWSCore: aws_config
     @test eltype(S3Array) == Zarr.ASCIIChar
     @test storagesize(S3Array) == 69
     @test String(S3Array[:]) == "Hello from the cloud!"
-  #end
+  end
 end
 
 @testset "GCS S3 Storage" begin
-  bucket = "cmip6"
-  store = "ScenarioMIP/DKRZ/MPI-ESM1-2-HR/ssp370/r4i1p1f1/Amon/tasmax/gn"
-  region = ""
-  aws_google = aws_config(creds=nothing, region="", service_host="googleapis.com", service_name="storage")
-  cmip6 = S3Store(bucket,store,aws = aws_google, listversion=1)
-  @test storagesize(cmip6) == 7557
-  @test Zarr.zname(cmip6) == "gn"
-  g = zopen(cmip6)
-  arr = g["tasmax"]
-  @test size(arr) == (384,192,1032)
-  @test eltype(arr) == Union{Missing, Float32}
-  @test all(isapprox.(arr[1:2,1:2,2], [237.519 239.618; 237.536 239.667]))
-  g2 = zopen("gs://cmip6/ScenarioMIP/DKRZ/MPI-ESM1-2-HR/ssp370/r4i1p1f1/Amon/tasmax/gn")
-  arr = g["tasmax"]
-  @test size(arr) == (384,192,1032)
-  @test eltype(arr) == Union{Missing, Float32}
-  @test all(isapprox.(arr[1:2,1:2,2], [237.519 239.618; 237.536 239.667]))
+  for backend in [Zarr.awssdk, Zarr.awss3]
+    bucket = "cmip6"
+    store = "ScenarioMIP/DKRZ/MPI-ESM1-2-HR/ssp370/r4i1p1f1/Amon/tasmax/gn"
+    region = ""
+    if backend == Zarr.awss3
+      # AWSS3 doesn't support service_host service_name
+      aws_google = aws_config(creds=nothing, endpoint="https://storage.googleapis.com")
```

Comment:
Note you can't set `service_host` with `AWSS3`, you have to set the `endpoint` instead.